### DISCUSSION TITLE
Why is access to the docker socket needed?

### DISCUSSION THREAD

**Discussion by ovizii** [bluet/docker-cronicle-docker#34] [Q&A] [answered]
I saw this in the docker-compsoe.yml file and was wondering what functionality it was needed for.

```
    volumes:
      - /var/run/docker.sock:/var/run/docker.sock
```

**Comment by bluet** [comment, marked as answer]
Hi @ovizii , as explained here https://github.com/bluet/docker-cronicle-docker/discussions/32#discussioncomment-6222812

It's for running nested container: docker-in-docker.
